nfafrontend/src/app/current-project/project-edit/project-edit.component.ts

```diff
@@ -76,6 +80,36 @@ export class ProjectEditComponent implements OnInit {
           );
         }}
 
+        if(project['stakeholderFactors']) {
+
+        stakeholderFactors = new FormArray([]);
+        for(const factor of project.stakeholderFactors) {
```

Comment:
Can't compile frontend, because:
> ERROR in src/app/current-project/project-edit/project-edit.component.ts(86,37): error TS2339: Property 'stakeholderFactors' does not exist on type 'Project'.

Please change **nfafrontend/src/app/shared/project.model.ts** accordingly or commit those changes if you forgot.

Reply:
Okay, I am not sure what happened, will try to fix.